src/browser/components/FileDrop/FileDrop.tsx

```diff
@@ -20,7 +20,8 @@
 import React, { useState } from 'react'
 import { connect } from 'react-redux'
 import { withBus } from 'react-suber'
-import SVGInline from 'react-svg-inline'
```

Comment:
Nice that you got rid of so many uses of `react-svg-inline`, looking at the it's [repo](https://github.com/MoOx/react-svg-inline) they say in big bold letters to use [svgr]( https://github.com/gregberge/svgr) instead. Out of scope for this PR but now when we only have one use of `react-svg-inline` in should be easy to get rid of in the future!